rtrlib/rtr_mgr.h

```diff
@@ -67,28 +68,37 @@ struct rtr_mgr_group {
 	enum rtr_mgr_status status;
 };
 
+//TODO Find a nicer way todo a linked list (without writing our own)
```

Comment:
I recommend to use the linked list from the linux kernel https://github.com/torvalds/linux/blob/master/include/linux/list.h.
Some small modifications needs to be done to make it work without the other kernel sources  but then it's awesome :-)